src/components/dev-hub/tab.js

```diff
@@ -41,38 +42,27 @@ const TabButton = styled('button')`
     ${({ isActive }) => (isActive ? activeStyles : defaultStyles)}
 `;
 
+const mapTabTextToButton = (textList, activeItem, handleClick) => {
+    return textList.map(item => {
+        const isActive = item === activeItem;
+        return (
+            <TabButton
+                key={item}
+                isActive={isActive}
+                onClick={() => handleClick(item)}
+            >
+                <P collapse>{item}</P>
+            </TabButton>
+        );
+    });
+};
```

Comment:
Super nit but if we are returning right away from a function (first thing we do is return), we can omit the `return` keyword and curly braces like so:

```
const mapTabTextToButton = (textList, activeItem, handleClick) => (
   textList.map(item => {
        const isActive = item === activeItem;
        return (
            <TabButton
                key={item}
                isActive={isActive}
                onClick={() => handleClick(item)}
            >
                <P collapse>{item}</P>
            </TabButton>
        );
    });
);
```

Just makes things a little cleaner!